lib/config/configuration.go

```diff
@@ -2041,6 +2051,23 @@ func applyWindowsDesktopConfig(fc *FileConfig, cfg *servicecfg.Config) error {
 	return nil
 }
 
+func staticHostsWithAddress(hosts []WindowsHost, defaultPort int) ([]servicecfg.WindowsHost, error) {
```

Comment:
Might as well remove the `defaultPort` argument since we always pass `defaults.RDPListenPort`.

If people want to use a different port, the answer won't be to change the default, but to specify the port in addr instead.